guides/common/modules/proc_configuring-project-settings-for-keycloak-authentication-using-the-cli.adoc

```diff
@@ -1,38 +1,40 @@
 [id="configuring-{project-context}-settings-for-keycloak-authentication-using-the-cli_{context}"]
-= Configuring {Project} settings for {keycloak-quarkus} authentication using the CLI
+= Configuring {Project} settings for {keycloak} authentication using the CLI
 
-Use this procedure to configure {Project} settings for {keycloak-quarkus} authentication using the {Project} CLI.
-
-Note that you can navigate to the following URL within your realm to obtain values to configure {Project} settings: `\https://{keycloak-example-com}/auth/realms/{Project}_Realm/.well-known/openid-configuration`
+If you are configuring a client that will provide Hammer CLI authentication, configure the {Project} client and add {keycloak} as an external authentication source.
 
 .Prerequisites
 
-* Ensure that the *Client authentication* setting in the {Project} client in the {keycloak-quarkus} web UI is set to OFF.
+* For {keycloak-quarkus}: Ensure that the *Client authentication* setting in the {Project} client in the {keycloak-quarkus} web UI is set to `OFF`.
```

Comment:
It's a bit weird to have separate sections for the 2 different variants and then suddenly still information of the other one showing up in the section (here for cli and web ui),
this is more like a note, not really blocking 🙂

Reply:
Addressed by introducing ifevals.